packages/react/src/card/card.tsx

```diff
@@ -280,6 +280,7 @@ export const Card = forwardRef<HTMLDivElement, CardProps>(
           { "hds-card--full-width": variant === "full-width" },
           { "hds-card--miniature": variant === "miniature" },
           { "hds-card--focus": variant === "focus" },
+          { "hds-card--slim": variant === "slim" },
```

Comment:
Variant needs to default to 'slim'. If not, the `hds-card__media` element will not get the proper margin-bottom.